README.md

```diff
@@ -249,7 +249,7 @@ end
 * `req.peer` - a Lua table with information about the remote peer
   (like `socket:peer()`).
 * `tostring(req)` - returns a string representation of the request.
-* `req:request_line()` - returns the request body.
+* `req:request_line()` - returns a first line of the http request packet (for example, `PUT /path HTTP/1.1`).
```

Comment:
Nit: I would replace 'the http request packet' with just 'the http request', because 'packet' is a TCP/IP level term or at least feels so.

Reply:
Nice! Excuse me for nits, but, please, squash the commits and force-push the resulting one.

Reply:
Thanks!